core/src/main/java/org/axonframework/eventhandling/EventProcessor.java

```diff
@@ -59,6 +61,15 @@
      */
     Registration registerInterceptor(MessageHandlerInterceptor<? super EventMessage<?>> interceptor);
 
+    /**
+     * Return the list of registered {@link MessageHandlerInterceptor}s for this event processor.
+     *
+     * @return the list of interceptors of this event processor
+     */
+    default Set<MessageHandlerInterceptor<? super EventMessage<?>>> getHandlerInterceptors() {
```

Comment:
This method should return a List, as interceptors have an order.

Reply:
Also, describe that the returned list is a live view (meaning changes are reflected in the returned list) and that it is read-only.